test/tools/jsontests/BlockChainTestsBoost.cpp

```diff
@@ -173,4 +173,35 @@ BOOST_AUTO_TEST_CASE(userDefinedFile)
 	dev::test::userDefinedTest(dev::test::doBlockchainTests);
 }
 BOOST_AUTO_TEST_SUITE_END()
+
+//BlockChainTests
+BOOST_AUTO_TEST_SUITE(BlockChainTestsRandom)
+
+BOOST_AUTO_TEST_CASE(bcRandom)
+{
+	using Bdit = boost::filesystem::directory_iterator;
+	std::string fillersPath = dev::test::getTestPath() + "/src/BlockchainTestsFiller/RandomTests";
+
+	Bdit iterator_tmp(fillersPath);
```

Comment:
this iterator can be declared inside `for`. Also don't use underscores in names.